ci/infra/testrunner/README.md

```diff
@@ -111,12 +114,29 @@ There are some arguments that are currently at the top level of the configuratio
 - nodeuser: the user name used to login into the platform nodes. Optional. 
 - ssh_key: specifies the location of the key used to access nodes. The default is to use the user's key located at `$HOME/.ssh/id_rsa`
 
+#### Packages
+The `packages` section configures the source of the packages to be installed in the nodes:
+
+* certificates: package with the certificates to be installed in the nodes in order to install other packages and images. For example, for installing SUSE self-signed packages:
+```
+packages:
+  certificates: "ca-certificates-suse"
+```
+* maintenance: repositories to be added to the installation repositories for installing maintenance updates. It takes the form of a map:
+```
+packages:
```

Comment:
What about 

```
packages:
    extra_repos:
        repo1: url/to/maintenance/repo1
        repo2: url/to/maintenance/repo2
```

I think the intended use case can be just exemplified like that and it is obvious to the user that the underlaying logic is just appending repositories at runtime time. Dos it make sense?

Reply:
It makes sense. I think this applies for both  `extra_repos` and `extra_packages`.